EventFilter/TotemRawToDigi/test/analyze_vfat_frames.py

```diff
@@ -0,0 +1,43 @@
+import FWCore.ParameterSet.Config as cms
+import FWCore.ParameterSet.VarParsing as VarParsing
+
+process = cms.Process("AnalyzeVFATFrames")
+
+# default options
+options = VarParsing.VarParsing ('analysis')
+options.inputFiles= 'file:/afs/cern.ch/user/j/jkaspar/public/run268608_ls0001_streamA_StorageManager.root',
```

Comment:
For sure no one in the "T" part of CTPPS has it. Where can I find more information about the /store/... ? Can we request some space there?

About the raw data: will they stay for long time in the /store/data/... ? In other words, is it save to use such path in config files? In general, where can I find more info on where CMS stores RAW data files?

Many thanks in advance!